modules/migration-using-must-gather.adoc

```diff
@@ -20,13 +28,33 @@ The `openshift-migration-must-gather-rhel8` image for {mtc-short} collects migra
 ----
 $ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v{mtc-version}
 ----
++
+The data is saved in a local `must-gather` directory.
+
+.Viewing metrics data with Prometheus
 
-. Remove authentication keys and other sensitive information.
-. Create an archive file containing the contents of the `must-gather` data directory:
+. Run the `oc adm must-gather` command with the `gather_metrics_dump` script:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image quay.io/konveyor/must-gather:latest -- /usr/bin/gather_metrics_dump
```

Comment:
There is a missing `=` after `--image`. Also, the image must be the correct one. So use the MTC must gather, not the MTV must gather image.